app/Mails/Smtp/MailerTemplateBladeResolver.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mails\Smtp;
+
+use Illuminate\Contracts\View\Factory as ViewFactory;
+
+final readonly class MailerTemplateBladeResolver implements MailerTemplateResolver
+{
+    private const TEMPLATE_ROOT = 'mails.';
+
+    /** @var string[]  */
+    private array $templates;
+
+    /** @var string[]  */
+    private array $subjects;
+
+    public function __construct(
+        private ViewFactory $view,
+    ) {
+        $this->templates = [
+            MailerTemplate::INTEGRATION_ACTIVATED->value => 'integration.activated',
+            MailerTemplate::ORGANISATION_UITPAS_REQUESTED->value => 'UdbOrganizer.requested',
+            MailerTemplate::ORGANISATION_UITPAS_APPROVED->value => 'UdbOrganizer.approved',
+            MailerTemplate::ORGANISATION_UITPAS_REJECTED->value => 'UdbOrganizer.rejected',
+        ];
+        $this->subjects = [
+            MailerTemplate::INTEGRATION_ACTIVATED->value => 'Je integratie {{ $integrationName }} is geactiveerd',
+            MailerTemplate::ORGANISATION_UITPAS_REQUESTED->value => 'Activatieaanvraag met integratie {{ $integrationName }} voor {{ $organizerName }}',
```

Comment:
This sounds like the activation of the integration itself, but that is something different. This is about requesting access to organisations.